Examples/Io/Root/include/ActsExamples/Io/Root/RootMeasurementWriter.hpp

```diff
@@ -209,6 +211,19 @@ class RootMeasurementWriter final : public WriterT<MeasurementContainer> {
       recBound[Acts::eBoundPhi] = fullVect[Acts::eBoundPhi];
       recBound[Acts::eBoundTheta] = fullVect[Acts::eBoundTheta];
       recBound[Acts::eBoundTime] = fullVect[Acts::eBoundTime];
+
+      Acts::BoundSymMatrix fullVar =
+          m.expander() * m.covariance() * m.expander().transpose();
```

Comment:
For my understanding: Should the second one not just be the projection?

Reply:
Ja, but it operates on a matrix.

Reply:
Ok, I just thought that .expander.transpose was be the same as .projection, with the latter being preferred